default/AI/MilitaryAI.py

```diff
@@ -16,6 +17,34 @@
 totMilRating = 0
 num_milships = 0
 verbose_mil_reporting = False
+best_ship_rating_cache = {}  # indexed by turn, value is rating of that turn
+
+
+def cur_best_mil_ship_rating(include_designs=False):
+    """Find the best military ship we have available in this turn and return its rating.
```

Comment:
`Find the best military ship we have available` it sounds for me that it is applicable only for existing ships.  But argument `include_designs` say that it can be different. 

At quick look I don't find any usage of `include_designs` may be remove it for current PR?

I really don`t like features that does not used. They are usually not well tested, they take place in code, they can be abandoned for years.  If you have plans on that argument  just add them as docstring or comments.